### ISSUE TITLE
Specto...Fork - Will not generate RealDebrid Authorize Code

### ISSUE THREAD

**Issue by steve02usa** [mrknow/filmkodi#920]
## Please follow the guide below 
- Put an "x" into all boxes [ ] relevant to your *issue*
- Use *Preview* tab to see how your issue will actually look like
---
## Kodi & Addon 
- Kodi verison: _____
- [ ] Filmy online - Mrknow
- [ ] MrknowTV
- [X ] Specto
- [ ] FanFILM
- Version /  2017.06.11.1
---
## Descryption / Opis
My version of Specto...Fork will not generate the RealDebrid activation code.  When I go into settings, Accounts, and click on the RealDebrid Activate Kodi field to generate the code, the screen clears, then comes back to the TV tab in settings.  No activationcode is ever generated.  I cannot use RealDebrid with Specto...Fork.  I have a RealDebrid account.  This works for Exodus and others.  Any assistance you can provide to help me resolve this problem would be greatly appreciated.  I have Uninstalled / Reinstalled Specto several times and still it does not work.
## Donation / Darowizna
This addons are free, but you can always thank the author for ... / Te wtyczki są darmowe, ale zawsze możesz podziękować autorowi za jej stworzenie...

[![paypal](https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif)](https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=HEV6TJY83S3Q4&lc=PL&item_name=Donation-Darowizna&currency_code=PLN&bn=PP%2dDonationsBF%3abtn_donateCC_LG%2egif%3aNonHosted)

**Comment by BankruptEd** [comment]
Nobody's is working as far as I know. Same for thevideo and openload.

**Comment by BankruptEd** [comment]
I wouldn't expect a fix anytime soon.

**Comment by uvxcoin** [comment]
Dish lawyers hard at work I see...

**Comment by BankruptEd** [comment]
Yup. It's cast a dark cloud over the Kodi world. This is an important case
that's being watched by many other players and other significant court
cases of late have been bad news. Hopefully it all blows over.

**Comment by FBVInvest** [comment]
confirm. can't add accounts either

**Comment by URMyBudE** [comment]
Ditto for me.  No response from the reauthorize link.  Other links on the accounts page work.

**Comment by spectofan** [comment]
Another confirmation. It is not working in Specto. Such a pity as Specto is by far the easiest video add-on to use imho. Until then it is back to Exo.....

**Comment by BankruptEd** [comment]
I am starting to believe he won't be back. One likes to think it's because
of the Dish court case, but that's all to do with live streams, so not
necessarily urgent for someone like Specto to disappear because of it, but
who knows. I think the time has come for developers to take greater caution
as far as their identities, and perhaps that's what all these disappeared
guys are doing; getting their identities secured. No one wants to end up
owing x number of dollars per infringement, for countless infringements.
That would be a bad day.
